terraform/global/um-pep-engine/config/pep-engine.tf

```diff
@@ -56,8 +53,11 @@ resource "kubernetes_service" "pep-engine" {
     labels = { app = "pep-engine" }
   }
 
+<<<<<<< HEAD
+=======
   depends_on = [null_resource.waitfor-login-service]
```

Comment:
This line might no longer be necessary thanks to:
module_depends_on = [module.um-login-service] on the main.tf file